draft-chfl-intarea-grim.xml

```diff
@@ -94,7 +94,33 @@ TBA
     <section>
       <name>Introduction</name>
 <t>
-TBA
+The determination of an optimal Maximum Transmission Unit (MTU) for end-to-end communication remains a persistent challenge even today.
```

Comment:
In this case actually "maximum packet size", but PMTU would be close.

Reply:
So any suggestions for the text?

Reply:
/Determining a suitable maximum packet size/

... or
/Determining a suitable maximum packet size (known as the Path MTU, PMTU)/

Reply:
Done, thank you!